VTEX - Antifraud Provider API.json

```diff
@@ -1950,57 +1504,56 @@
                 "properties": {
                     "id": {
                         "type": "string",
-                        "default": "D3AA1F",
-                        "description": "VTEX transaction ID. The ID of the transaction related with this payment."
+                        "description": "VTEX transaction ID. The ID of the transaction related with this payment.",
+                        "example": "D3AA1FC8372E430E8236649DB5EBD08E"
                     },
                     "reference": {
                         "type": "string",
-                        "default": "v32vtx-01",
-                        "description": "VTEX order reference key. The key of the order (from VTEX OMS system) related with this payment."
+                        "description": "VTEX order reference key. The key of the order (from VTEX Orders system) related with this payment.",
+                        "example": "vtexargentina"
                     },
                     "value": {
                         "type": "integer",
-                        "format": "int32",
-                        "default": 10,
-                        "description": "VTEX transaction order value."
+                        "description": "VTEX transaction order value.",
+                        "example": 10
                     },
                     "ip": {
                         "type": "string",
-                        "default": "10.0.0.1",
-                        "description": "The original ip address from browser."
+                        "description": "Original IP address from browser.",
+                        "example": "10.0.0.1"
                     },
                     "store": {
                         "type": "string",
-                        "default": "insert store name here",
-                        "description": "VTEX has a main Name for the store. This unique name is often used in url to access VTEX configurations."
+                        "description": "VTEX has a main name for the store. This unique name is often used in url to access VTEX configurations.",
```

Comment:
qual URL? é o account name usado para acessar o admin?

Reply:
Modificado para: "Store name. This is the same name used to access the store Admin via URL."